resources/valueset-qicore-present-on-admission.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<ValueSet xmlns="http://hl7.org/fhir">
+  <id value="qicore-present-on-admission"/>
+  <meta>
+    <profile value="http://hl7.org/fhir/StructureDefinition/shareablevalueset"/>
+  </meta>
+  <url value="http://hl7.org/fhir/us/qicore/ValueSet/qicore-present-on-admission"/>
+  <version value="4.0.0"/>
+  <name value="QICorePresentOnAdmission"/>
+  <title value="QICore Present On Admission Codes"/>
+  <status value="draft"/>
+  <experimental value="false"/>
+  <date value="2021-05-14"/>
+  <publisher value="Health Level Seven, Inc. - CQI WG"/>
+  <contact>
+    <telecom>
+      <system value="url"/>
+      <value value="http://hl7.org/special/committees/CQI"/>
+    </telecom>
+  </contact>
+  <description value="Value Set for QICore Present On Admission."/>
+  <jurisdiction>
+    <coding>
+      <system value="urn:iso:std:iso:3166"/>
+      <code value="US"/>
+      <display value="United States of America"/>
+    </coding>
+  </jurisdiction>
+  <compose>
```

Comment:
Consider the way it is done in https://build.fhir.org/ig/HL7/UTG/branches/UP-219-clone-create-present-on-admission-indicators-value-set/ValueSet-POAIndicators.json.html that is:
```
  "compose": {
    "include": [
      {
        "system": "https://www.cms.gov/Medicare/Medicare-Fee-for-Service-Payment/HospitalAcqCond/Coding"
      }
    ],
    "exclude": [
      {
        "system": "https://www.cms.gov/Medicare/Medicare-Fee-for-Service-Payment/HospitalAcqCond/Coding",
        "concept": [
          {
            "code": "1"
          }
        ]
      }
    ]
  }
```
Not clear if this code system is case sensitive, but cms.gov link uses upper case.

Reply:
Initially created the valueset using \<exclude> but the output in the publisher generated the following:

![Screen Shot 2021-08-05 at 4 00 16 PM](https://user-images.githubusercontent.com/40005056/128414419-6eece533-28f6-4cf2-8554-324c542bb416.png)

Thus generated it by including all of the codes instead of using \<exclude and output via publisher was the following:

![Screen Shot 2021-08-05 at 3 57 18 PM](https://user-images.githubusercontent.com/40005056/128414818-0b8ad18f-7b1f-40f1-bf34-f6d92922c2b1.png)

Reply:
> Not clear if this code system is case sensitive, but cms.gov link uses upper case.

http://build.fhir.org/ig/HL7/UTG/CodeSystem-presentOnAdmission.json.html shows `"caseSensitive": false,`